File: 0001-two-sum/solution.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const arr = [];
    upper:
    for (let i = 0; i < nums.length; i++) {
        for (let n = 0; n < nums.length; n++) {
            if (n==i) continue;
            if (target == nums[i] + nums[n]) {
            arr.push(i, n);
            break upper;
        }
        
        }
    }

    return arr;
};

File: 0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        m = {} # number : index

        for i, num in enumerate(nums):
            diff = target - num
            if diff in m:
                return [i, m[diff]]
            
            m[num] = i
